orchagent/orchdaemon.cpp

```diff
@@ -83,7 +83,7 @@ void OrchDaemon::start()
         Selectable *s;
         int fd, ret;
 
-        ret = m_select->select(&s, &fd, 1);
```

Comment:
this timeout allows the merging time for the m_toSync map to merge and allows the time for items with unresolved dependencies to be resolved without retrying too many times